client/me/security-2fa-initial-setup/index.jsx

```diff
@@ -35,10 +48,54 @@ class Security2faInitialSetup extends Component {
 					) }
 				</p>
 
+				<FormFieldset>
+					<FormRadio
+						id="app-auth"
+						name="auth_method"
+						value="app-based"
+						defaultChecked
+						onChange={ this.setAuth }
+					/>
+					<FormLabel htmlFor="app-auth">
+						<Gridicon icon="phone" />
+						<span className="security-2fa-initial-setup__item-title">
+							{ this.props.translate( 'Set up using an app', {
+								context: 'A label used during Two-Step setup.',
+							} ) }
+						</span>
+					</FormLabel>
+					<FormSettingExplanation>
+						{ this.props.translate(
+							'Use an application on your phone to get two-step authentication codes when you login.'
+						) }
+					</FormSettingExplanation>
+				</FormFieldset>
+				<FormFieldset>
+					<FormRadio
+						id="sms-auth"
+						name="auth_method"
+						value="sms-settings"
+						onChange={ this.setAuth }
+					/>
+					<FormLabel htmlFor="sms-auth">
+						<Gridicon icon="chat" />
+						<span className="security-2fa-initial-setup__item-title">
+							{ this.props.translate( 'Set up using SMS', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Reply:
Resolved in fbdbdfa16ab525cc6a4f90470594869c5c7ed5bd